### PR TITLE
Add a top-level pom.xml that collects the submodules

### PR DESCRIPTION
**Pull request by elharo**
@netdpb

### CODE REVIEW

**Comment by elharo** [comment]
FYI dashboard tests require setting maven.home like so:

$ mvn -Dmaven.home=/usr/local/Cellar/maven/3.5.0 clean test

This becomes moot once we use our own upgrade checker and fix #41

**Comment by elharo** [comment]
Ping ping ping

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+               http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud.tools.opensource</groupId>
+  <artifactId>parent</artifactId>
```
[the remark this answers is not in the document]
Comment (elharo) [reply pom.xml]: Suggestions? Maybe cloud-tools-opensource-parent?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    <module>dashboard</module>
+  </modules>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.guava</groupId>
+        <artifactId>guava</artifactId>
+        <version>${guavaVersion}</version>
+      </dependency>
```
[the remark this answers is not in the document]
Comment (elharo) [reply pom.xml]: These are the three that are currently shared between the projects.

File: pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (garrettjonesgoogle) [reply pom.xml]: I think that would work fine.

File: pom.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (garrettjonesgoogle) [reply pom.xml]: Ok that makes sense.

**Pull request merged**